db/models.py

```diff
@@ -1 +1,41 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.SET_NULL,
+        null=True
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(
+        "description provided by a user about himself/herself",
+        max_length=255
+    )
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.SET_NULL,
+        null=True
+    )
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/225003184-c8cfbe4a-3dfb-45e5-bb0a-0cc1cd1b695d.png)
player should always have race